microsoft-edge/toc.yml

```diff
@@ -143,8 +143,8 @@
               items:
                 - name: Overview
                   href: devtools-guide-chromium/customize/index.md
-                - name: Dark theme
-                  href: devtools-guide-chromium/customize/dark-theme.md
+                - name: Theme
```

Comment:
Should this start with a verb to match other pages in this section ("Change placement" and "Match browser language")? If so, it could be "Apply a theme" or "Change theme."